portfolio/src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -1,10 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I don't think you can remove this. It's used as build tool in your pom.xml
Did you try running the application after removing this?

Reply:
I ran it before pushing. Is it not working? 
Also, I didn't change anything outside the webapp directory.

Reply:
Try removing your target folder (portfolio/target) and running it again. It needs _appengine-web.xml_ to be copied to target for deployment. If you remove it from _src/main/webapp/WEB-INF/_. Then it can not be copied during the first execution.
This is the App Engine's configuration file. This file tells Google Cloud how to deploy your code.

Reply:
I have updated the appengine-file.